gensim/models/hdpmodel.py

```diff
@@ -464,7 +464,10 @@ def update(self, corpus):
         """
         save_freq = max(1, int(10000 / self.chunksize))  # save every 10k docs, roughly
         chunks_processed = 0
-        start_time = time.clock()
+        try:
+            start_time = time.process_time()
+        except AttributeError:
+            start_time = time.clock()
```

Comment:
Both seem wrong to me. Why should user case about the process time? Who needs such runtime constraints?

What people care about is the wallclock time, so perhaps `time.time()`.

Reply:
Thank you for the review!
As you pointed out, I should have used `time.time()`.
I'll fix it.